template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Extensions/AzureAd/AzureAdOptions.cs

```diff
@@ -1,18 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Microsoft.AspNetCore.Authentication.Extensions
+﻿namespace Microsoft.AspNetCore.Authentication.Extensions
 {
     public class AzureAdOptions
     {
         public string ClientId { get; set; }
         public string ClientSecret { get; set; }
-        public string AzureAdInstance { get; set; }
+        public string Instance { get; set; }
```

Comment:
I don't have a strong opinion, we should use whatever the official term for this is, my main intent was to drop the name of of the options from the property